backend/test/autoplay.clj

```diff
@@ -1,12 +1,13 @@
 (ns autoplay
   (:require [configs.hands :as hands]
+            [configs.rows :as rows]
             [api.base :as api]
             [rules.create-game :as create-game]
             [rules.play-card :as play-card]))
 
 (defn as-api
   [strategy1 strategy2]
-  (let [game (api/create-game)
+  (let [game (api/create-game {:limits rows/limitless})
```

Comment:
wait, isn't `row/limitless` the default? I still like it for readability, but now I question if I've understood

Reply:
Right now `rows/limitless` and `rows/default` are exactly the same. `rows/default-limits` can be changed without any problem (within backend). When frontend handles the limits, we will be able to change `rows/default-limits` without having to touch any test.